src/App.js

```diff
@@ -1,17 +1,50 @@
 import React, { useState } from "react"
 import './App.css';
 import Button from './component/Button';
+import ToDoForm from "./component/TodoForm";
 
+function App() {
 
 
+  const [clickedButton, setClickedButton] = useState(false)
+  const[todos, setTodo] = useState([])
+ 
 
+  const handleClick =  (buttonId) => {
+     setClickedButton(buttonId);
+  }
+
+ 
+  const addTodo = (newTodo) => {
+    setTodo([...todos, newTodo])
+}
 
-function App() {
+
+   
+    return (
+      <div className="App">
+        <div className='app-container'>
+          <h1>TODO App</h1>
+            <div className='button-container'>
+               <Button color="#8888b9" label="Todo" onClick={() => {handleClick("button1")}}  isClicked={clickedButton === "button1"}  />
+               <Button color="#8888b9" label="Todo list" onClick={() => {handleClick("button2")}}  isClicked={clickedButton === "button2"} />
+            </div>
+           
+            {clickedButton=='button1' && (
```

Comment:
`clickedButton=='button1'` this is reused in some places. So, declare a variable for this and reuse it everywhere.

eg:
```
const isTodoButtonClicked = clickedButton=='button1';

  {isTodoButtonClicked && <ToDoForm onAddTodo={addTodo} />}
```